prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh

```diff
@@ -196,7 +196,8 @@ metadata:
     component: knative-eventing-channel-kafka
     kyma-project.io/installation: ""
 stringData:
-  kafka.brokers: ${K8S_SECRET_BROKER}
+  kafka.brokers.hostname: ${K8S_SECRET_BROKER}
```

Comment:
I think it is clean to use the following to have some consistency:
```
kafka.brokers.hostname: ${K8S_SECRET_BROKER_HOSTNAME}
kafka.brokers.port: ${KAFKA_SECRET_BROKER_PORT}
```

Reply:
@sayanh Done